### ISSUE TITLE
how can i make the makefile look inside src/

### ISSUE THREAD

**Issue by gabriel-fondato** [educ8s/Raylib-CPP-Starter-Template-for-VSCODE#3]
i just like to be organized and i want to know now can i modify the makefile to search for cpp files inside a folder named src

**Comment by mmmovania** [comment]
> i just like to be organized and i want to know now can i modify the makefile to search for cpp files inside a folder named src

This is how I did it. 
[Makefile.txt](https://github.com/educ8s/Raylib-CPP-Starter-Template-for-VSCODE/files/14126593/Makefile.txt)

All my source files (main.cpp, *,cpp, *.h) are in the src subfolder on the root. I made a obj folder at the same level as src folder to store intermediate obj files. The makefile is at the main root folder. 

See if this helps :)
